src/test/kotlin/LottoNumberTest.kt

```diff
@@ -0,0 +1,19 @@
+package lotto
+
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class LottoNumberTest {
+    @Test
+    fun `throw error exception when number is out of the range between 1 and 45`() {
+        for (i in 1..45) {
+            LottoNumber(i)
+        }
```

Comment:
This for-loop in a test is not asserting anything.
If this object gets created without any exceptions, let's assert that with our test library 👍